webview/src/plots/components/PlotsContainer.tsx

```diff
@@ -137,11 +136,25 @@ export const PlotsContainer: React.FC<PlotsContainerProps> = ({
     </div>
   )
 
+  const isTooltip = (el: Element) => {
+    let currentNode = el
+    while (currentNode.nodeName !== 'SUMMARY') {
```

Comment:
just to be safe - should we check here for null/undefined (whatever it is in JS :) ). I mean, let's break the cycle if we got to the very top and still don't see summary.

Reply:
This function is called for the `onClick` inside of the `<summary />` element. The `<summary />` element is always the top node of this event. I'll add the `BODY` node name in the unexplained case it would go pass the `SUMMARY` one as it doesn't hurt, but I'm totally sure this can't happen.